lang/java/pom.xml

```diff
@@ -472,7 +472,6 @@
         <groupId>joda-time</groupId>
         <artifactId>joda-time</artifactId>
         <version>${joda.version}</version>
-        <optional>true</optional>
```

Comment:
This shouldn't remove the optional element anymore, I fixed the code but not the POM. The dependency is optional.